coderd/database/migrations/000264_update_forgot_password_notification.up.sql

```diff
@@ -0,0 +1,10 @@
+UPDATE notification_templates
+SET
+    title_template = E'Reset your password for Coder',
+    body_template = E'Hi {{.UserName}},\n\nA request to reset the password for your Coder account has been made.\n\nIf you did not request to reset your password, you can ignore this message.',
```

Comment:
> If you did not request to reset your password, you can ignore this message.
Did this language come from a developer or product?

Reply:
Good question! I just refactored the body template using the previous verbiage, but let's check with @stirby.

Reply:
I think we can simplify.

```
Hi {{.UserName}},\n\nUse the link below to reset your password.\n\nIf you did not make this request, you can ignore this message.

```